{
    "has_execution_error": true,
    "execution_output": "Fatal Python error: init_sys_streams: can't initialize sys standard streams\nPython runtime state: core initialized\nOSError: [Errno 9] Bad file descriptor\n\nCurrent thread 0x00000001dcac5c40 (most recent call first):\n  <no Python frame>\n",
    "pred": {
        "error": "malformed_output"
    },
    "gold": {
        "itinerary": [
            {
                "action": "meet",
                "person": "Karen",
                "start_time": "09:13",
                "end_time": "10:28"
            },
            {
                "action": "meet",
                "person": "Laura",
                "start_time": "11:45",
                "end_time": "12:45"
            },
            {
                "action": "meet",
                "person": "Deborah",
                "start_time": "13:05",
                "end_time": "14:50"
            },
            {
                "action": "meet",
                "person": "Steven",
                "start_time": "15:03",
                "end_time": "17:03"
            },
            {
                "action": "meet",
                "person": "Jason",
                "start_time": "17:08",
                "end_time": "18:38"
            },
            {
                "action": "meet",
                "person": "Elizabeth",
                "start_time": "18:44",
                "end_time": "19:59"
            },
            {
                "action": "meet",
                "person": "Carol",
                "start_time": "21:30",
                "end_time": "22:30"
            }
        ]
    },
    "status": "Execution error: malformed_output",
    "violated_constraint": {},
    "is_exact_match": false,
    "constraints_satisfied": false,
    "pass_number": 5
}